emr.links.action.yml

```diff
@@ -0,0 +1,23 @@
+emr.type_add:
```

Comment:
As discussed, let's add some consistency to the yml files with regards to namespacing. It gets very complicated to understand what they are